src/components/network-chart/index.js

```diff
@@ -26,15 +26,15 @@ import type {
   Milliseconds,
   UnitIntervalOfProfileRange,
 } from '../../types/units';
-import type { PreviewSelection } from '../../types/actions';
 import type {
   ExplicitConnectOptions,
   ConnectedProps,
 } from '../../utils/connect';
 
 require('./index.css');
 
-const ROW_HEIGHT = 16;
+const ROW_HEIGHT = 17.5;
+const CONTAINER_WIDTH = 150 / window.screen.width * 100; // this will be replaced by the viewport
```

Comment:
The `CONTAINER_WIDTH` is the ratio of
- width of the thread label column in percent
- width of screen

We need a ratio / percent value as `_getMarkerPosition` as well as `_getMarkerLength` work with `ms` and the common sense here is the ratio in %.

For further info, please see the comments in those 2 functions.